README.md

```diff
@@ -77,16 +86,12 @@ Processes an input string using [Markdown](https://markdown-it.github.io).
 
 Output: `<p>A paragraph with some <em>emphasis</em>.</p>\n`
 
-#### slug
+#### slug (deprecated)
 
 Processes an input string by lowercasing it, replacing whitespace with hyphens, and stripping special characters to
 create a URL-safe version.
 
-```nunjucks
-{{ "Here’s my title!" | slug }};
-```
-
-Output: `heres-my-title`
+**NOTE: This filter is deprecated as of eleventy-plugin-fluid 1.0. Instead, use Eleventy's new `slugify` filter.**
```

Comment:
Is there a link to slugify documentation that can be added here?

Reply:
Done in [5652053](https://github.com/fluid-project/eleventy-plugin-fluid/pull/109/commits/5652053652cd9cb3a326dd526b28fb7ea4ca3dac).